### DISCUSSION TITLE
connect nodes on touch devices

### DISCUSSION THREAD

**Discussion by moklick** [xyflow/xyflow#1021] [New Features]
Add the possibility to connect nodes via clicks/taps. For this we could add a `connectOnClick` option that is true by default for touch devices.

**Comment by Immortalin** [comment]
https://rete.js.org/ works perfectly on mobile.

**Reply by leejustin** [reply]
Thanks @Immortalin . I opted to use React Flow in the end after vetting different libraries. `rete`'s node connections on mobile seem decent but aren't "perfect" as it can be easy for my fat fingers to pan the screen instead of landing on a connection point. It's probably something that I can fix for a better user experience but the investment didn't seem to be worth it.  I ended up building some tooling to programmatically create nodes and edges. The feature would still be nice though.

**Comment by karocksjoeli** [comment]
any update on this ? 
possible workaround is appreciated

**Comment by CarloCattano** [comment]
+1 , Im new to the library , but will try to implement this soonish.
Need to find the place in the code where the click listener is and try to implement a touch event instead . 
It already works for moving around objects and such , so it should not be hard I hope.

**Comment by bluebill1049** [comment]
Thanks! ❤️  This would be an amazing feature to add! I am excited about this.
